src/commands/configure-docker-credentials-store.yml

```diff
@@ -0,0 +1,43 @@
+description: >
+  Configure a credentials store for docker to use.
+  See: https://docs.docker.com/engine/reference/commandline/login/#credentials-store#credentials-store
+  Supported platforms: Linux and macOS.
+
+parameters:
+  helper-name:
+    description: >
```

Comment:
Maybe here we could switch this to an enum of "MacOS" or "Linux". Or an enum of the currently supported helpers.

Or this could be inferred as below